Alcatraz/Helpers/ATZShell.m

```diff
@@ -77,8 +77,11 @@ - (void)setUpTerminationHandlerForTask:(NSTask *)task completion:(void(^)(NSStri
             if (task.terminationStatus == 0) {
                 completion(output, nil);
             } else {
-                NSString* reason = [NSString stringWithFormat:@"Task exited with status %d", task.terminationStatus];
-                completion(output, [NSError errorWithDomain:reason code:666 userInfo:@{ NSLocalizedDescriptionKey: reason }]);
+                const NSUInteger maxLength = 500;
+                NSString *truncatedOutput = [[output substringToIndex:MIN(maxLength, output.length)] stringByAppendingString:output.length > maxLength ? @"…" : @""];
```

Comment:
I improved this in 835fe4e7b866b2c0e6d4cfc72f9d95c8aa76d4b7. Also, the git shell command outputs are always logged regardless of failure.